integration_tests/conftest.py

```diff
@@ -59,3 +63,18 @@ def cluster(request, cronos, geth):
         yield geth
     else:
         raise NotImplementedError
+
+@pytest.fixture(scope="module")
+def worker_index(worker_id):
+    match = re.search(r"\d+", worker_id)
+    return int(match[0]) if match is not None else 0
+
+@pytest.fixture(scope="module")
+def cronos_cluster(worker_index, pytestconfig, tmp_path_factory):
+    "default cluster fixture"
+    yield from cluster_fixture(
```

Comment:
I think you can use the default `cronos` fixture?

Reply:
It is ok to use `cronos` fixture to create multiple nodes. But I found pytest-xdist is also working alright. Just curious about why we abandon `pytest-xidst` in cronos?

Reply:
`pytest-xidst` is for running tests in parallel to save time, the current test running time in Cronos is ok I think.
several issue with `pytest-xdist` based on previous experience:
- fixtures can't be shared in multiple worker processes, so each work processes need to setup it's own cronos devnet, while only a single one is needed in normal case.
- Not easy to distribute the workload evenly, because different test cases could rely on different setup, so in chain-main we distribute test cases by module, so the spetial devnet only need to set up once in one of the worker processes, but the running times of different test module might vary a lot. But distributing test cases by functions might end up with too many duplicated devnet setups in the worker processes.
- The github action exec env get stressed when there are many parallel executions, results in timeout errors.